addons/quickmount/ACE_Settings.hpp

```diff
@@ -36,6 +36,6 @@ class ACE_Settings {
         description = CSTRING(PriorityDescription);
         isClientSettable = 1;
         force = 0;
-        values[] = {"Driver", "Gunner", "Commander", "Passenger"};
```

Comment:
It is vanilla translations? But I can't see it translation in my game.

Image
![alt text](http://i.imgur.com/FJ6NfnE.jpg)